### ISSUE TITLE
PHPUnit 6 support

### ISSUE THREAD

**Issue by ruudk** [mybuilder/phpunit-accelerator#11]
It uses namespaces.

**Comment by b1rdex** [comment]
@gavinlove ping. This is implemented in #12, I'm using it few monthes with no bugs.

**Comment by gavinlove** [comment]
I should get time to look over the weekend but since the tests are failing I can't merge the pull.

**Comment by b1rdex** [comment]
@gavinlove sorry didn't noticed that. Failing error is
```
PHP Fatal error:  Declaration of MyBuilder\PhpunitAccelerator\TestListener::addError(PHPUnit\Framework\Test $test, Exception $e, $time)
must be compatible with PHPUnit_Framework_TestListener::addError(PHPUnit_Framework_Test $test, Exception $e, $time)
in /home/travis/build/mybuilder/phpunit-accelerator/src/TestListener.php on line 10
```
Phpunit <6 has namespaced compability layer, but it's not not complete and namespaced `TestListener` was introduced only in 5.7: https://github.com/sebastianbergmann/phpunit/blob/5.7/src/ForwardCompatibility/TestListener.php

So it looks like there is only way to resolve migration to phpunit 6 — release new major version with phpunit 5.7+ support only.

**Comment by b1rdex** [comment]
@gavinlove ping.

**Comment by MaartenMortier** [comment]
ping, need this too..

**Comment by enumag** [comment]
Needed this too. Eventually I rewrote the listener and removed useless stuff like ignorePolicy too. This is the result: https://gist.github.com/enumag/26b49b14ac50069db8b402b4c3b3789d.

**Comment by b1rdex** [comment]
@gavinlove I opened #13 that fixes problems from #12 and is more legacy compatible (>=5.7).
All tests are passing, please take a look on it.

**Comment by b1rdex** [comment]
@ruudk @MaartenMortier @enumag version 2.0 was just released with phpunit 5.7/6+ support — https://github.com/mybuilder/phpunit-accelerator/releases

**Comment by gavinlove** [comment]
#13 Fixed this
